scripts/produceLKG.ts

```diff
@@ -90,7 +90,7 @@ async function copyFromBuiltLocal(fileName: string) {
 }
 
 async function copyFilesWithGlob(pattern: string) {
-    const files = glob.sync(path.join(source, pattern)).map(f => path.basename(f));
+    const files = glob.sync(path.join(source, pattern).replace(/\\/g, "/")).map(f => path.basename(f));

```

Comment:
Maybe just use `path.posix.join` instead?

Reply:
```
> path.posix.join("C:\\Users\\Jack\\", "src/**/*.ts")
'C:\\Users\\Jack\\/src/**/*.ts'
```

It does not work 🤔

Reply:
I see. Maybe the calls to glob shouldn't be messing with the paths at all, and should instead pass the path as the cwd option (then pass the pattern in unchanged).

Reply:
@Jack-Works did you still have time to do this, or would you like me to edit this PR / send a new one?

Reply:
yes, I can try it